README.md

```diff
@@ -164,5 +164,31 @@ This project can be packaged as either a noarch rpm or an srpm.
 $ make rpm                # Or, `make srpm`
 ```
 
+### i18n
+
+For a general guide on how to write translatable strings, see [weldr.io](http://weldr.io/Translating-welder-web-strings/)
```

Comment:
Have you posted this article yet? I'm not able to find it on weldr.io

Reply:
Not yet, it's in _drafts. It's viewable here, with slightly worse formatting: https://github.com/weldr/weldr.github.io/blob/master/_drafts/Translating-welder-web-strings.md

Reply:
This article is great!